### PR TITLE
[EZP-27369] Integrate ez-design-engine to ezplatform.com

### PR DESCRIPTION
**Pull request by lolautruche**
> https://jira.ez.no/browse/EZP-27369

* Moved templates
* Changed config to use `@ezdesign` namespace
* Use `ezdesign` asset package for eligible assets.
* Changed usage of `include` tag to `include` function.

Note that `ezdesign` package was not used for CSS as Assetic is in use, and it is not possible to use themes with Assetic.

### CODE REVIEW

**Comment by lolautruche** [comment]
Ping @bdunogier @SylvainGuittard

**Comment by bdunogier** [comment]
Does anyone have comments or questions about this integration, guys ? Even if we don't merge it now, it is still a fairly important addition :)

File: composer.json [hunk 1]
```diff
@@ -43,7 +43,8 @@
         "ezsystems/content-on-the-fly-prototype": "~0.1.7",
         "willdurand/js-translation-bundle": "^2.6.4",
         "ezsystems/comments-bundle": "~6.0",
-        "knplabs/packagist-api": "~1.3"
+        "knplabs/packagist-api": "~1.3",
+        "ezsystems/ezplatform-design-engine": "^1.0@rc"
```
Comment (andrerom) [inline composer.json]: can be stable now

**Comment by andrerom** [comment]
Well it seems like it probably should be merged sooner rather than later as it will continue to get more conflicts if not :|

My only question by now would be what `with_context = false` does.

**Comment by andrerom** [comment]
And for assets, maybe lets plan to move to sf 3 and webpack encore once we can, and see how ezdesign can work with that.

**Comment by bdunogier** [comment]
I'll take care of updating this to the latest.
Nonetheless, I'm expecting at least one review from somebody in the webteam.

**Comment by SylvainGuittard** [comment]
We wanted first update ezplatform.com to 1.10 last week before merging this PR.
@damianz5 can take care of the conflicts next week and also update the new templates for the bundle list.

File: .gitignore [hunk 2]
```diff
@@ -54,3 +54,4 @@ composer.lock
 behat.yml
 .php_cs.cache
 auth.json
+ide-twig.json
```
Comment (SylvainGuittard) [inline .gitignore]: What do you think about adding `ide-twig.json` to `ezplatform` repo instead?

**Comment by lolautruche** [comment]
@andrerom with_context=false prevents included templates to get caller context (e.g variables) and thus enforces explicit variables passing

**Comment by damianz5** [comment]
Conflicts resolved, I've updated the code: `with_context=false` has the same result as `only`

**Comment by lolautruche** [comment]
+1

**Comment by SylvainGuittard** [comment]
Thanks @lolautruche for this great contribution

**Pull request merged**